### PR TITLE
Added uniform vector functions

### PR DESCRIPTION
**Pull request by autrilla**
This doesn't compile yet, as some bindings are missing on gleam. I'll PR there and squash when it's merged.

### CODE REVIEW

**Comment by autrilla** [comment]
Waiting on https://github.com/servo/gleam/pull/75

File: src/webgl.rs [hunk 1]
```diff
@@ -336,20 +352,36 @@ impl WebGLCommand {
                 gl::link_program(program_id),
             WebGLCommand::Uniform1f(uniform_id, v) =>
                 gl::uniform_1f(uniform_id, v),
+            WebGLCommand::Uniform1fv(uniform_id, v) =>
+                gl::uniform_1fv(uniform_id, v.as_slice()),
```
Comment (emilio) [inline src/webgl.rs]: I think the `&v` syntax is more common (it dereferences automatically `&Vec<T>` to `&[T]`).

Comment (autrilla) [inline src/webgl.rs]: Indeed, updated

**Comment by emilio** [comment]
`r=me` with that change:

@bors-servo: delegate+

**Pull request merged**